src/config/cron.md

```diff
@@ -1,13 +1,28 @@
 # Cron
 
-Void Linux comes without a default cron daemon, you can choose one of multiple
-cron implementations, by installing the package and enabling the system service.
+[cron](https://en.wikipedia.org/wiki/Cron) is a daemon for running programs at
+regular intervals. The programs and intervals are specified by using
+[crontab(1)](https://man.voidlinux.org/crontab.1) to edit a `crontab` file.
```

Comment:
`a crontab(5) file`?

Reply:
Unfortunately, not all implementations install a `crontab(5)` man page; `dcron` (which i use) certainly doesn't. And the exact things allowed by crontabs can vary from impl. to impl.

Reply:
Oh, I see :c 

Fair enough, then.

Reply:
Oh and also, even when there's such a file, it's not necessarily literally called `crontab`: cf. `fcrontab`, `incrontab`.